pkgs/by-name/cr/cryptpad/package.nix

```diff
@@ -59,6 +61,17 @@ let
     }
   ];
 
+  x2t_version = "v7.3+1";
+  x2t = fetchurl {
```

Comment:
How about using fetchzip here?

Reply:
@martinetd can we address this before merging ?

Reply:
Ultimately it's (fetchzip and copy) or (fetchurl and unzip), both are two steps so I don't have any strong feeling here (fetchurl and unzip is actually slightly more efficient if you consider the build process, but not enough to matter anyway and I can agree with fetchzip+copy looking simpler)

Either way it's not my PR, you probably meant to ask @jfly -- letting him reply :)

Reply:
Perhaps ideally we'd use `fetchzip` and create a symlink, but we're also creating that `.version` file in the `X2T_DIR`.

I'd prefer to leave this as is. `fetchurl` + `unzip` closest to what [upstream's script does](https://github.com/cryptpad/cryptpad/blob/2024.9.1/install-onlyoffice.sh#L192).

(I'm going to explore a completely different approach in a subsequent PR. I'm sort of hoping we can figure out a way to reuse upstream's install script.)

Reply:
> I'm going to explore a completely different approach in a subsequent PR. I'm sort of hoping we can figure out a way to reuse upstream's install script

That worked out! Please check out: https://github.com/NixOS/nixpkgs/pull/365514